### ISSUE TITLE
nimble_operator rule is not triggered for `beTrue()`

### ISSUE THREAD

**Issue by Igor-Palaguta** [realm/SwiftLint#2613]
### Describe the bug

nimble_operator rule is not triggered for `beTrue()` Nimble's predicate

### Environment

* SwiftLint version (run `swiftlint version` to be sure)?
0.30.1
* Installation method used (Homebrew, CocoaPods, building from source, etc)?
package file from github
* Paste your configuration file:

```yml
opt_in_rules:
  - nimble_operator
```

* Which Xcode version are you using (check `xcode-select -p`)?
Version 10.0 (10A255)
* Do you have a sample that shows the issue?

`echo "expect(success).to(beTrue())" | swiftlint lint --no-cache --use-stdin --enable-all-rules`

```swift
// This triggers a violation:
expect(success).to(beTrue())

// This does not trigger a violation:
expect(success) == true
```

**Comment by marcelofabri** [comment]
I *think* it was a conscious decision when I first wrote that rule because I personally preferred to use `to(beTrue())` at that time.

But we should catch this though. PRs are welcome!

**Comment by marcelofabri** [comment]
Implemented in #2622.
